test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestThreadMXBean.java

```diff
@@ -70,20 +70,24 @@ public class TestThreadMXBean {
 		ignoredAttributes.add("ObjectName");
 		attribs = new Hashtable<String, AttributeData>();
 		attribs.put("AllThreadIds", new AttributeData("[J", true, false, false));
+		attribs.put("CurrentThreadAllocatedBytes", new AttributeData(Long.TYPE.getName(), true, false, false));
```

Comment:
The test doesn't assert that each of these attributes exist: I think it should.

However, even with this change, `CurrentThreadAllocatedBytes`, `ThreadAllocatedBytes` and `ThreadAllocatedMemoryEnabled` don't exist. `ThreadAllocatedBytes` should not be expected to exist based on the behavior of Temurin-17+35.

Reply:
Fixed.

Reply:
Your test changes verify that `MBeanInfo.getAttributes()` includes `CurrentThreadAllocatedBytes` and `ThreadAllocatedMemoryEnabled`, but my testing shows that `MBeanServer.getAttributes()` 
can't be used to get those two attributes.

Reply:
That's because those APIs throw UnsupportedOperationException with OpenJ9. Getting the attributes from the server includes calling the APIs and getting the values.

Reply:
Ok, that makes sense.